openassessment/xblock/submission_mixin.py

```diff
@@ -240,7 +256,7 @@ def _get_download_url(self):
             logger.exception("Error retrieving download URL.")
             return ''
 
-    def _get_student_item_key(self):
+    def _get_student_item_key(self, content_type=None):
```

Comment:
Seems like you aren't using the content_type parameter, and no one is passing it.